### PR TITLE
use webpack instead of buble

### PR DESCRIPTION
**Pull request by karissa**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const path = require('path')
+
+module.exports = (_, argv) => ({
+  entry: path.normalize(`${__dirname}/src/renderer/main.js`),
+  target: 'electron-main',
+  output: {
+    path: path.normalize(`${__dirname}/static`),
+    filename: 'bundle.js',
```
Comment (ralphtheninja) [inline webpack.config.js]: `static/bundle.js` :+1:

Comment (karissa) [reply webpack.config.js]: are you saying to change `bundle.js` to `static/bundle.js`

Comment (karissa) [reply webpack.config.js]: seems to work, the `path` seems to be properly taking care of the directory

Comment (ralphtheninja) [reply webpack.config.js]: No, I'm saying it was a good thing to do.

**Comment by Jikstra** [comment]
Thanks! :)

File: package.json [hunk 2]
```diff
@@ -8,14 +8,15 @@
...
+    "dist": "npm run build:prod && electron-builder --publish=onTagOrDraft",
     "dev": "npm run build && npm run start",
-    "build": "npm run build-es5 && npm run build-conversations",
+    "build": "npm run build-conversations && npm run build:dev",
     "build-conversations": "npm run build-conversations-ts && npm run build-conversations-css",
     "build-conversations-ts": "cd conversations && tsc",
     "build-conversations-css": "cd conversations && node-sass stylesheets/manifest.scss build/manifest.css",
-    "build-es5": "buble src --output es5"
+    "build:dev": "webpack --mode=development --progress --profile --colors",
+    "build:prod": "webpack --mode=production --profile --colors && npm run build-conversations"
```
Comment (ralphtheninja) [inline package.json]: This order needs to change as well?

File: package.json [hunk 3]
```diff
@@ -8,14 +8,15 @@
     "test": "standard && node test/unit",
     "test-integration": "node test/integration",
     "start": "electron . --debug --disable-http-cache",
-    "pack": "npm run build && electron-builder --dir",
-    "dist": "npm run build && electron-builder --publish=onTagOrDraft",
+    "pack": "npm run build:prod && electron-builder --dir",
+    "dist": "npm run build:prod && electron-builder --publish=onTagOrDraft",
     "dev": "npm run build && npm run start",
-    "build": "npm run build-es5 && npm run build-conversations",
+    "build": "npm run build-conversations && npm run build:dev",
```
Comment (ralphtheninja) [inline package.json]: Remove this one. Lets just keep `build:dev` and `build:prod`.

Comment (ralphtheninja) [reply package.json]: (So `.travis.yml` needs to be updated to do `npm run build:prod`)

**Comment by ralphtheninja** [comment]
Something broke the integration tests. Not sure what has changed there.

**Comment by ralphtheninja** [comment]
```
[path] external "path" 42 bytes {main} [built]
    [./src/renderer/main.js] 284ms -> [./src/localize.js] 77ms -> factory:170ms building:43
8ms dependencies:0ms = 969ms
    + 468 hidden modules

> deltachat-desktop@1.7.4 start /home/lms/src/DELTACHAT/deltachat-desktop
> electron . --debug --disable-http-cache

App threw an error during load
Error: Cannot find module './es5/main'
    at Module._resolveFilename (module.js:543:15)
    at Function.Module._resolveFilename (/home/lms/src/DELTACHAT/deltachat-desktop/node_mod
ules/electron/dist/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:473:25)
    at Module.require (module.js:586:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/lms/src/DELTACHAT/deltachat-desktop/index.js:1:156)
    at Object.<anonymous> (/home/lms/src/DELTACHAT/deltachat-desktop/index.js:3:3)
    at Module._compile (module.js:642:30)
    at Object.Module._extensions..js (module.js:653:10)
    at Module.load (module.js:561:32)
A JavaScript error occurred in the main process
Uncaught Exception:                                                                        
Error: Cannot find module './es5/main'                                                     
    at Module._resolveFilename (module.js:543:15)                                          
```

@karissa Try removing your `es5` folder

File: package.json (hunk 2 above)
Comment (karissa) [reply package.json]: ok simplified it

File: package.json (hunk 3 above)
Comment (karissa) [reply package.json]: just got rid of `prod`, doesn't really matter too much for us because we aren't shipping this to a client

**Comment by karissa** [comment]
ok @ralphtheninja thanks for the review

File: package.json [hunk 4]
```diff
@@ -8,14 +8,14 @@
     "test": "standard && node test/unit",
     "test-integration": "node test/integration",
     "start": "electron . --debug --disable-http-cache",
-    "pack": "npm run build && electron-builder --dir",
-    "dist": "npm run build && electron-builder --publish=onTagOrDraft",
+    "pack": "npm run build:prod && electron-builder --dir",
+    "dist": "npm run build:prod && electron-builder --publish=onTagOrDraft",
```
Comment (ralphtheninja) [inline package.json]: `build:prod` -> `build`

**Comment by ralphtheninja** [comment]
@karissa Nice work buddy!

**Pull request merged**